trading-hello-service-api/build.sbt

```diff
@@ -0,0 +1,13 @@
+// Nexus configuration
+publishTo := {
+  val nexus = "https://nexus3.softwaremill.com/"
+  if (isSnapshot.value)
+    Some("snapshots" at nexus + "repository/scala-academy-snapshots/")
+  else
+    Some("releases"  at nexus + "repository/scala-academy-release/")
```

Comment:
Perhaps the repository name should be `scala-academy-releases` (plural)?

Reply:
renamed